lectures/delta-time/README.md

```diff
@@ -0,0 +1,360 @@
+<h2 align=center>Lecture 05</h2>
+
+<h1 align=center>Delta Time</h1>
+
+<h3 align=center> Imperial Year CCXXIV</h3>
+
+***Song of the day***: _[**ハネムーン (Honeymoon)**](https://www.youtube.com/watch?v=It1IucO2wfw) by LOLUET (feat. 理芽 [RIM]) (2024)._
+
+### Sections
+
+1. [**Matrix Operations Review**](#part-1-matrix-operations-review)
+2. [**Spaces**](#part-2-spaces)
+3. [**Timing, FPS, and Delta Time**](#part-3-timing-fps-and-delta-time)
+
+### Part 1: _Matrix Operations Review_ 
+
+Recall our three transformations, with their respective matrices:
+
+- **Scaling**:
+
+![scal-mult-ex](assets/scal-mult-ex.svg)
+
+<sub>**Figure 1**: A matrix being scaled by a scalar value of 2.</sub>
+
+- **Rotation**:
+
+![scal-scal](assets/rot-mat-ex.svg)
+
+<sub>**Figure 2**: A column vector being rotated by an angle of Θ.</sub>
+
+- **Translation**:
+
+![scal-trans](assets/translation-mat.svg)
+
+<sub>**Figure 3**: The standard transformation matrix for the x- and y-dimensions.</sub>
+
+Thus far, we have been applying these transformations to the model matrix one after the other:
+
+```c++
+model_matrix = glm::translate(model_matrix, glm::vec3(TRANS_VALUE, 0.0f, 0.0f));
```

Comment:
Add `g_` to `model_matrix` to match the source code. Also, I think `TRANS_VALUE` is called `TRAN_VALUE` in the source code.